ChangeLog

```diff
@@ -1,3 +1,58 @@
+* Fri Oct 29 2021 Ken Gaillot <kgaillot@redhat.com> Pacemaker-2.1.2-rc1
+- Changesets: 448
+- Diff: 221 files changed, 16502 insertions(+), 11737 deletions(-)
+
+- Features added since Pacemaker-2.1.1
+  + build: when built with --with-initdir, Pacemaker uses the value to find
+           LSB resources (in addition to being where Pacemaker's own
+           init scripts are installed)
+  + build: cmocka is new dependency for unit tests ("make check")
+  + rpm: fence_watchdog now comes with pacemaker package (not pacemaker-cli)
+  + daemons: metadata for cluster options supports OCF 1.1 standard
+  + executor: nagios warning results now map to OCF "degraded" result code
+  + fencing: pcmk_delay_base can optionally specify different delays per node
+  + fencing: pcmk_host_map supports escaped characters such as spaces in values
+  + resources: HealthIOWait agent supports OCF 1.1 standard, and validate works
+  + tools: crm_mon shows exit reasons for actions failed due to internal errors
+  + tools: crm_mon failed action display is more human-friendly by default
+  + tools: crm_resource --force-* now outputs exit reasons if available
+
+- Fixes since Pacemaker-2.1.1
+  + tools: fix crm_mon --hide-headers and related options
+           (regression introduced in 2.0.4)
```

Comment:
I don't know how consistent we've been with this, but when I did the 2.0.5 release, I put things like this into their own header:
```
- Fixes for regressions introduced in Pacemaker-2.0.4
   + tools: fix crm_mon --hide-headers and related options
```

Reply:
The format hasn't been consistent, but at least we've gotten better about calling out regressions in some way. This is just rc1, so I'll think about it for the next one :)